src/AspNet.Security.OAuth.Moodle/MoodleAuthenticationOptions.cs

```diff
@@ -37,9 +37,12 @@ public MoodleAuthenticationOptions()
             ClaimActions.MapJsonKey(ClaimTypes.MobilePhone, "phone1");
             ClaimActions.MapJsonKey(ClaimTypes.GivenName, "firstname");
             ClaimActions.MapJsonKey(ClaimTypes.Surname, "lastname");
+            ClaimActions.MapJsonKey(ClaimTypes.AuthenticationMethod, "auth");
```

Comment:
Please extend the test and HTTP bundle to validate these three new claims are present.

Reply:
Tests are ready now.